tests/res_diff_tests/pass_fix_write_compare_errors.ion

```diff
@@ -0,0 +1,22 @@
+{
+ bad: 	{},
+ good: {
+    "test": {
+        'ion-java_1': {
+            result: FAIL,
+             write_compare: {
+                errors: ErrorReport::[
+                    {
+                      message: "test",
+                    }
+                ]
+             }
+        },
+        'ion-java_2': {
+            result:FAIL
```

Comment:
Did you mean for this to be `result:PASS`? If not, we should probably have a file that tests that case.

Reply:
Added more tests.